functions/boardWithRandomlyPlacedShips .ts

```diff
@@ -1,18 +1,13 @@
 /**Places ships from randomly generated coordinates that is not outside the grid or already occupied on any provided 10*10 grid*/
 const boardWithRandomlyPlacedShips = (
   shipLength: number,
-  Board: number[][],
+  board: number[][],
 ): number[][] => {
   const width = 10;
   const orientation =
     Math.round(Math.random()) >= 0.5 ? "horizontal" : "vertical";
-  console.log("running");
 
-
-  const updatedBoard = [...Board];
```

Comment:
This is the same as using `board`, as spread operator copies items by reference.